README.md

```diff
@@ -1,12 +1,3 @@
-<p align="center">
-  <a href="https://developers.mymoid.com" target="_blank" rel="noopener noreferrer">
-   <picture>
-      <source media="(prefers-color-scheme: dark)" srcset="./examples/nextjs/public/m.svg">
-      <img src="./examples/nextjs/public/m.svg" height="64">
```

Comment:
Please keep it. I uploaded the file to:

`https://mymoid-static.s3.amazonaws.com/images/mymoid-suit/mymoid_m.svg`

Checks this out.